src/fidesops/models/fidesops_user.py

```diff
@@ -14,6 +14,8 @@ class FidesopsUser(Base):
     username = Column(String, unique=True, index=True)
     hashed_password = Column(String, nullable=False)
     salt = Column(String, nullable=False)
+    last_login_at = Column(DateTime(timezone=True), nullable=True)
```

Comment:
Great start! Now we just need to have this field update every time the user logs in (https://github.com/ethyca/fidesops/blob/main/src/fidesops/api/v1/endpoints/user_endpoints.py#L89)